visual-dotnet/SauceLabs.Visual.IntegrationTests/LoginPageConcurrent.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using OpenQA.Selenium.Remote;
+
+namespace SauceLabs.Visual.IntegrationTests;
+
+[Parallelizable(ParallelScope.Children)]
+[IntegrationTest(skipIfSingle: true)]
+public class LoginPageConcurrent
+{
+    private VisualClient? VisualClient { get; set; }
+
+    [OneTimeSetUp]
+    public async Task Setup()
+    {
+        VisualClient = await VisualClient.Create(Utils.GetSauceRegion());
```

Comment:
```suggestion
        VisualClient = await VisualClient.Create();
```

what do you think about using a default constructor?

Reply:
Updated.